lib/vector-tap/Cargo.toml

```diff
@@ -7,15 +7,22 @@ publish = false
 license = "MPL-2.0"
 
 [dependencies]
+async-graphql = { version = "7.0.7", default-features = false, features = ["playground"], optional = true}
 colored = { version = "2.1.0", default-features = false }
+futures = { version = "0.3.30", default-features = false, features = ["std"] }
+glob = { version = "0.3.1", default-features = false }
```

Comment:
Could these be turned into workspace dependencies?

Reply:
Turned `futures` and `glob` into workspace dependencies, async-graphql is optional so kept as is

Reply:
Up to you, but workspace dependencies [can be optional too](https://github.com/vectordotdev/vector/blob/696ead1019b8a31b453285e2936fa4de9ed73f9b/Cargo.toml#L156).